bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/GhostScienceAreaGeometry.scala

```diff
@@ -0,0 +1,19 @@
+package edu.gemini.spModel.gemini.ghost
+
+import java.awt.Shape
+import java.awt.geom.Ellipse2D
+
+import edu.gemini.spModel.inst.ScienceAreaGeometry
+import edu.gemini.spModel.obs.context.ObsContext
+import scalaz._
+import Scalaz._
+
+object GhostScienceAreaGeometry extends ScienceAreaGeometry {
+  private val size = 7.5 * 60 // arcmin
+  private val radius = size / 2
+
+  /** Create the shape for the science area based on the instrument configuration.
+   * This shape is not adjusted for position angle or offsets. */
+  override def unadjustedGeometry(ctx: ObsContext): Option[Shape] =
+    new Ellipse2D.Double(-radius, -radius, size, size).some
```

Comment:
Is this the only use of `scalaz`? Maybe we should reduce our reliance on scalaz and do `Some(...)` instead

Reply:
I guess I'm not too concerned about it because `scalaz` is scattered throughout the codebase already.

Reply:
I'm indifferent either way. I don't really like Scalaz so I'm find with `Some(_)` instead of `.some`, but yeah, it is all over the code base as is.

Reply:
I guess I'm being influenced by the need to port these to newer versions of scala java :D Feel free to ignore